pkg/agent/services/service.go

```diff
@@ -16,27 +16,40 @@ const (
 	device  = "device"
```

Comment:
Why do we have folder services, and in it `service.go`? Are these local services on the gateway?

It is a little confusing, because we have `service.go` which relates to agent and then `services` which relate to local services...

Reply:
But I currently have no better idea for naming.

Reply:
can we rename `service.go` to `agent.go`
or maybe `services/info.go`

Reply:
Well... I am more bothered about the folder name `services`. I think we should rename `service.go` into `agent.go` for sure, and we can leave this `services` for now, but we need to document it somewhere - maybe add one README.md in the `agent` or these `services` folder...

Reply:
for the folder maybe `info`, or `stats` so we have `info/service.go`
or maybe since it is basically only for heartbeat to call it `heartbeat/service.go`